actions/admin/access_keys.go

```diff
@@ -52,22 +53,22 @@ func (a *Action) accessKeysSearch(c *gin.Context) {
 // @Description	Access Keys Count
 // @Tags		Admin
 // @Produce		json
-// @Param		CountAccessKeys body CountAccessKeys false "Enables filtering of elements to be counted"
+// @Param		CountAccessKeys body filter.CountAccessKeys false "Enables filtering of elements to be counted"
```

Comment:
as above - shouldn't we point to the specific structure: `filter.AdminCountAccessKeys`?

Reply:
Yes. fixed 👍